pkg/ebpftracer/c/tracee.bpf.c

```diff
@@ -6823,19 +7001,24 @@ int BPF_KPROBE(tty_open, struct inode *inode, struct file *filep)
         return 0;
     }
 
-    if (!should_submit(TTY_OPEN, p.event)) {
-        return 0;
+    if (should_submit(TTY_WRITE, p.event)) {
+        unsigned long ino = BPF_CORE_READ(inode, i_ino);
+        u32 one = 1;
+        bpf_map_update_elem(&tty_opened_files, &ino, &one, BPF_NOEXIST);
```

Comment:
When switching the map to `BPF_LRU_HASH` I would switch to `BPF_ANY`. Otherwise we would keep older `fd`s in the map, which are less likely to trigger a write IMHO.